cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -149,7 +149,7 @@ func Run(c *cloudcontrollerconfig.CompletedConfig, stopCh <-chan struct{}) error
 	if c.SecureServing != nil {
 		unsecuredMux := genericcontrollermanager.NewBaseHandler(&c.ComponentConfig.Generic.Debugging, checks...)
 		handler := genericcontrollermanager.BuildHandlerChain(unsecuredMux, &c.Authorization, &c.Authentication)
-		if err := c.SecureServing.Serve(handler, 0, stopCh); err != nil {
+		if _, err := c.SecureServing.Serve(handler, 0, stopCh); err != nil {
```

Comment:
do we need a TODO to wire up waits for these other server processes, or are we not planning to wait for requests to complete for non-apiservers?

Reply:
A TODO maybe, but I don't anybody will add that any time soon. Those APIs are not that critical that it's worth it.